src/xml_comment.cc

```diff
@@ -109,7 +109,7 @@ void
 XmlComment::Initialize(v8::Handle<v8::Object> target)
 {
     NanScope();
-    v8::Local<v8::FunctionTemplate> t = NanNew<v8::FunctionTemplate>(New);
+    v8::Local<v8::FunctionTemplate> t = NanNew<v8::FunctionTemplate>(static_cast<NAN_METHOD((*))>(New));
```

Comment:
Yes, you are right about that. It cannot tell which one is meant to be used, as both are in the same namespace, although it would not make sense to use the method that does not fit the signature. I'll see if this can be improved somehow.

On a side note, that other New method does not seem to be used at all. Removing it produces no errors.